docs/ros/ros/installation/robot.mdx

```diff
@@ -0,0 +1,127 @@
+
+ROS 2 Humble uses [Ubuntu 22.04](https://releases.ubuntu.com/jammy/) as its Tier 1 operating system. Though other operating systems are supported,
+it is highly recommended to use Ubuntu 22.04. For the robot PC, Ubuntu 22.04 Server should be installed. The server version of the OS will not install a
+graphical user interface (GUI).
+
+### Networking
+
+Once the operating system is installed, the robot PC needs to be connected to the internet. TODO.
+
+### ROS 2 Humble
+
+Follow the [official instructions](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html) for installing ROS 2 Humble from debian packages.
+
+### Clearpath Package Server
+
+The Clearpath package server needs to be added to the robot PC to access robot packages.
+
+Configure Ubuntu's APT package manager to add the package server:
+
+**1. Install the authentication key for the packages.clearpathrobotics.com repository**
+
+```
+wget https://packages.clearpathrobotics.com/public.key -O - | sudo apt-key add -
+```
+
+**2. Add the debian sources for the repository** 
+
+
+```
+sudo sh -c 'echo "deb https://packages.clearpathrobotics.com/stable/ubuntu $(lsb_release -cs) main" > /etc/apt/sources.list.d/clearpath-latest.list'
+```
+
+**3. Update your computer’s package cache**
+
+
+```
+sudo apt-get update
+```
+
+### Clearpath Robot metapackage
+
+Once the Clearpath package server has been added, the [Clearpath Robot](https://github.com/clearpathrobotics/clearpath_robot) metapackage can be installed.
+
+```
+sudo apt install ros-humble-clearpath-robot
+```
+
+Installing this package will also install all dependencies required to run any Clearpath robot and supported sensors, other than Clearpath firmware packages.
+
+### Firmware Update
+
+All platforms with the exception of the A200 will require a firmware update to run ROS 2 Humble.
+
+#### J100
+
+Jackal firmware can be flashed from the Jackal's PC.
+
+**1. Download the Jackal Firmware package from the Clearpath package server**
+
+```
+sudo apt-get update
+sudo apt-get install ros-humble-jackal-firmware
+```
+
+**2. Place the Jackal MCU into bootloader mode**
+
+Switch the *PWR_MODE* switch from *NORM* to *ALT*. If the robot is on, press the *M_RST* button. Otherwise, turn the
+robot on with the power button.
+
+**3. Flash the firmware**
+
+```
+jackal_firmware_flash.sh
+```
+Wait for the firmware to be flashed.
+
+:::caution
+Do not turn the robot off or unplug the micro-USB cable while flashing firmware.
+:::
+
+**4. Place the Jackal MCU into normal mode**
+
+Once the firmware has been flashed, place the Jackal MCU back into normal mode by switching the PWR_MODE switch from ALT to NORM.
+The robot will power off. Turn the robot back on by pressing the power button.
+
+### Setup folder
+
+Create the setup folder where the [robot.yaml](../config/yaml) file will be stored, and where files will be generated:
+
+```
+sudo mkdir /etc/clearpath/ -p && sudo chmod 666 /etc/clearpath/
```

Comment:
Should we have one of our debian packages create this folder instead? And possibly even install an empty version of robot.yaml that just has some comments?

Reply:
That's a good idea. I can add it to `clearpath_robot`.